<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test_widget</class>
 <widget class="QWidget" name="test_widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#test_widget {
background-color:qradialgradient(spread:pad, cx:0.483682, cy:0.307, radius:0.5, fx:0.478, fy:0, stop:0 rgba(173, 185, 207, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,2,0,0">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item alignment="Qt::AlignHCenter|Qt::AlignTop">
    <widget class="QLabel" name="countdown_label">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Countdown</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="typingtext_label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Here is example text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="results_label">
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="again_button">
       <property name="text">
        <string>Again</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="return_button">
       <property name="text">
        <string>Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
